website/docs/guides/container_service_extension_4_0_cluster_management.html.markdown

```diff
@@ -0,0 +1,272 @@
+  - `VCD_WORKER_PLACEMENT_POLICY`: Name of an existing VM Placement Policy. Can be empty (`""`)
+  - `VCD_WORKER_STORAGE_PROFILE`: Name of an existing Storage Profile, for example `"*"` to use the default.
+  - `DISK_SIZE`: Specifies the storage size for each node (VM). It uses the [same units as every other Kubernetes resource](https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/),
+    for example `"1Gi"` to use 1 gibibyte (1024 MiB), or `"1G"` for 1 gigabyte (1000 MB).
+  - `VCD_CATALOG`: The catalog where the TKGm OVAs are. For example, in the above CSE installation it was `tkgm_catalog`.
+  - `VCD_TEMPLATE_NAME` = The TKGm OVA name, for example `ubuntu-2004-kube-v1.22.9+vmware.1-tkg.1-2182cbabee08edf480ee9bc5866d6933`
+  - `POD_CIDR`: The CIDR used for Pod networking, for example `"100.96.0.0/11"`.
+  - `SERVICE_CIDR`: The CIDR used for Service networking, for example `"100.64.0.0/13"`.
+
+  There are three additional variables that we added manually: `TKR_VERSION` and `TKGVERSION`.
+  To know their values, you can use [this script](https://github.com/vmware/cluster-api-provider-cloud-director/blob/main/docs/WORKLOAD_CLUSTER.md#script-to-get-kubernetes-etcd-coredns-versions-from-tkg-ova),
+  or check the following table that gives some script outputs for you:
+
+| OVA name                                                 | TKR_VERSION               | TKGVERSION |
+|----------------------------------------------------------|---------------------------|------------|
+| v1.19.16+vmware.1-tkg.2-fba68db15591c15fcd5f26b512663a42 | v1.19.16---vmware.1-tkg.2 | v1.4.3     |
+| v1.20.14+vmware.1-tkg.2-5a5027ce2528a6229acb35b38ff8084e | v1.20.14---vmware.1-tkg.2 | v1.4.3     |
+| v1.20.15+vmware.1-tkg.2-839faf7d1fa7fa356be22b72170ce1a8 | v1.20.15---vmware.1-tkg.2 | v1.5.4     |
+| v1.21.8+vmware.1-tkg.2-ed3c93616a02968be452fe1934a1d37c  | v1.21.8---vmware.1-tkg.2  | v1.4.3     |
+| v1.21.11+vmware.1-tkg.2-d788dbbb335710c0a0d1a28670057896 | v1.21.11---vmware.1-tkg.2 | v1.5.4     |
+| v1.22.9+vmware.1-tkg.1-2182cbabee08edf480ee9bc5866d6933  | v1.22.9---vmware.1-tkg.1  | v1.5.4     |
+| v1.22.13+vmware.1-tkg.2-ea08b304658a6cf17f5e74dc0ab7544f | v1.22.13---vmware.1-tkg.1 | v1.6.1     |
+| v1.21.14+vmware.2-tkg.5-d793afae5aa18e50bd9175e339904496 | v1.21.14---vmware.2-tkg.5 | v1.6.1     |
+| v1.23.10+vmware.1-tkg.2-b53d41690f8742e7388f2c553fd9a181 | v1.23.10---vmware.1-tkg.1 | v1.6.1     |
+
+In [the TKGm cluster creation example][cluster], the built-in Terraform function `templatefile` is used to substitute every placeholder
+mentioned above with its final value. The returned value is the CAPVCD YAML payload that needs to be set in the `capi_yaml` placeholder in the
+JSON template.
+
+~> Notice that we need to replace `\n` with `\\n` to avoid breaking the JSON contents when the YAML is set, and also `\"` to `\\\"` for
+the same reason. This is also done in [the mentioned example][cluster].
```

Comment:
Maybe this should be more clear why this needs to be set? Maybe just say that the line breaks would not be interpreted correctly